infra/variables.tf

```diff
@@ -0,0 +1,79 @@
+variable "project_name" {
+    type = string
+    default = "poc"
```

Comment:
Is there a way to eliminate the duplication between this value and the one in the tfvars file?